tests/e2e/client/apiproxy_bookstore_test.py

```diff
@@ -54,20 +55,25 @@ def _send_request(self, path, api_key=None,
                 print 'Negative test: remove auth.'
                 r = self._call_http(path, api_key, None, data, method)
                 self.assertEqual(r.status_code, 401)
-                self.assertEqual(
-                    r.text,
-                    'Jwt is missing')
+                try:
+                    rpcStatus = json.loads(r.text)
+                    self.assertEqual(rpcStatus, json.loads('{"message":"Jwt is missing","code":401}'))
+                except:
+                    print("fail to decode json payload")
```

Comment:
should we call self.assertXXX to mark the test is failed.  Just printing some error will not mark the test as failured. 
Same for line 76